tst/unit/test_pararrays.cpp

```diff
@@ -487,8 +553,10 @@ TEST_CASE("Check registry pressure", "[ParArrayND][performance]") {
 
   // view of views. See:
   // https://github.com/kokkos/kokkos/wiki/View#6232-whats-the-problem-with-a-view-of-views
+  using view_3d_t =
+      Kokkos::View<Real ***, parthenon::LayoutWrapper, parthenon::DevMemSpace>;
```

Comment:
should these types be stored somewhere globally?

Reply:
I would lean towards no. I just put this there so I didn't have to type as much, but I don't think it would be used many places in the code and the type name is probably not descriptive enough if it is used far away from where it is defined.

Reply:
:+1: